gunicorn/http/wsgi.py

```diff
@@ -158,7 +158,7 @@ def create(req, sock, client, server, cfg):
     if isinstance(client, string_types):
         environ['REMOTE_ADDR'] = client
     elif isinstance(client, binary_type):
-        environ['REMOTE_ADDR'] = str(client)
+        environ['REMOTE_ADDR'] = client.decode()
```

Comment:
I usually prefer specifying the encoding explicitly in a codebase that should support both Python 2 and Python 3 since the default encoding is different between the two versions, but in this case I don't think `client` can contain non-ASCII values in Python 2.